mock/mock.go

```diff
@@ -661,7 +661,7 @@ func (m *Mock) AssertNumberOfCalls(t TestingT, methodName string, expectedCalls
 			actualCalls++
 		}
 	}
-	return assert.Equal(t, expectedCalls, actualCalls, fmt.Sprintf("Expected number of calls (%d) does not match the actual number of calls (%d).", expectedCalls, actualCalls))
+	return assert.Equal(t, expectedCalls, actualCalls, "Expected number of calls (%d) does not match the actual number of calls (%d).", expectedCalls, actualCalls)
```

Comment:
If we're doing this I'd also change this to assert.Equalf as we're advising people not to use the msgAndArgs variadic.